config_test.yaml

```diff
@@ -12,15 +13,26 @@ path :
   model_oracle_MLP : "./storage/model_oracle/oracle_MLP"
 
 env:
-  main : "aptamers"
-  dict_size : 4
-  min_len : 10
-  max_len : 20
-  min_word_len : 1
-  max_word_len : 1
+  main : "grid" #grid aptamers
+  aptamers:
+    dict_size : 4
+    min_len : 10
+    max_len : 20
+    min_word_len : 1
+    max_word_len : 1
+  grid:
+    n_dim: 2
+    length: 3
+    min_step_length: 1
+    max_step_length: 1
+    cell_min: -1
+    cell_max: 1
+  reward_beta: 1.0
+  reward_norm: 1.0
```

Comment:
reward_beta and reward_norm could environment (grid or aptamers) specific ?

Reply:
Yes I agree. The config file is kinda random right now. It could definitely be more organised